metadatastore/core_v0.py

```diff
@@ -986,7 +987,8 @@ def find_descriptors(start_col, start_cache,
     _format_time(kwargs, tz)
 
     col = descriptor_col
-    event_descriptor_objects = col.find(kwargs)
+    event_descriptor_objects = col.find(kwargs,
```

Comment:
This PR adds a 'time' index for descriptors, so these can be returned in a deterministic order, which helps migration validation.